t/openssl/ssl/ssl_client.t

```diff
@@ -53,6 +53,7 @@ our $ClientContentBy = qq{
 no_long_string();
 
 env_to_nginx("CI_SKIP_NGINX_C");
+env_to_nginx("FORCE_AUX_FUNCS");
```

Comment:
use 
```suggestion
env_to_nginx("CI_SKIP_KONG_SSL_FUNCS");
```
lua code would need to be changed too

Reply:
changed.